### DISCUSSION TITLE
经过项目leader同意，建立TG交流群，欢迎大家加入

### DISCUSSION THREAD

**Discussion by darvenking** [linweiyuan/go-chatgpt-api#197] [General]
发广告会被移出群聊

telegram群：[https://t.me/welcome_chatgpt](https://t.me/welcome_chatgpt)

**Comment by xiaoguaishou92** [comment]
过期了。。错过机会

**Reply by xbl916** [reply]
是哦 我26号看到这个项目，二维码已经过期了

**Reply by gclm** [reply]
能帮忙拉一下吗？

**Reply by xbl916** [reply]
我的意思是我26号才发现这个项目，二维码的有效期是到25号。。。所以我也没进去

**Reply by darvenking** [reply]
我现在更新下

**Reply by yanqing8** [reply]
超过两百人，只能邀请了

**Reply by nowhereman1999** [reply]
确实，所以老哥你进群了吗，能拉一下吗
